pkg/registry/core/service/rest_test.go

```diff
@@ -1235,40 +1235,6 @@ func TestServiceRegistryExternalTrafficGlobalBeta(t *testing.T) {
 	}
 }
 
-// Validate that ExternalTraffic is default to Global for loadBalancer service.
-func TestServiceRegistryExternalTrafficDefaultGlobal(t *testing.T) {
```

Comment:
Yep, service created through storage.Create() would not be defaulted. Defaulting should happen in later phases, and that is tested in default_test.go